src/sentry/snuba/discover.py

```diff
@@ -1122,10 +1122,17 @@ def find_histogram_min_max(fields, min_value, max_value, user_query, params, dat
                 first_quartile = row[q1_alias]
                 third_quartile = row[q3_alias]
 
-                if first_quartile is not None and third_quartile is not None:
-                    interquartile_range = abs(third_quartile - first_quartile)
-                    upper_outer_fence = third_quartile + 3 * interquartile_range
-                    fences.append(upper_outer_fence)
+                if (
+                    first_quartile is None
+                    or third_quartile is None
+                    or math.isnan(first_quartile)
+                    or math.isnan(third_quartile)
```

Comment:
These `nan` are coming from quantiles on empty sets correct? There isn't any chance of an `inf` or `-inf` here right?

Reply:
Yes, with the right filters, the `quantile` is computed on the empty set resulting in `NaN`. I don't believe there to be any way for this to result in `inf` or `-inf`. Though to be safe, we can guard against it with `math.isfinite` instead. Would that be preferred?

Reply:
Nope, this is perfect as-is IMO :+1: just wanted to confirm